### PR TITLE
Provisionnaly fix JSON parsing bug on spotify import

### PR DESCRIPTION
**Pull request by sim0mo**
This PR, if applied, will fix a small bug when parsing playlist that contain non-JSON-parsable data. It will have to be done in a more elegant way one of these days.

### CODE REVIEW

File: app/src/main/java/com/github/fribourgsdp/radio/ImportSpotifyPlaylistsActivity.kt [hunk 1]
```diff
@@ -22,6 +24,7 @@ class ImportSpotifyPlaylistsActivity : AppCompatActivity() {
         setContentView(R.layout.activity_import_spotify_playlists)
         val userToken = intent.getStringExtra("auth_token")
         TOKEN = userToken
+        Log.println(Log.INFO, "***", TOKEN.toString())
```
Comment (VictorSchneuwly) [inline app/src/main/java/com/github/fribourgsdp/radio/ImportSpotifyPlaylistsActivity.kt:27]: Do we need the `print` on the `Log` to stay for `master`?

File: app/src/main/java/com/github/fribourgsdp/radio/ImportSpotifyPlaylistsActivity.kt [hunk 2]
```diff
@@ -113,17 +116,19 @@ class ImportSpotifyPlaylistsActivity : AppCompatActivity() {
...
                     val songs = parsedResponse.getJSONArray("items")
                     for (i in 0 until songs.length()){
-                        var songObject = songs.getJSONObject(i).getJSONObject("track")
+                        val songObject =
+                        try {
+                            songs.getJSONObject(i).getJSONObject("track")
+                        } catch (e : JSONException){
+                            // TODO: handle error
+                            continue
+                        }
```
Comment (VictorSchneuwly) [inline app/src/main/java/com/github/fribourgsdp/radio/ImportSpotifyPlaylistsActivity.kt:129]: Do you plan on doing this in a later sprint or did you forget about it?

File: app/src/main/java/com/github/fribourgsdp/radio/ImportSpotifyPlaylistsActivity.kt (hunk 1 above)
Comment (sim0mo) [reply app/src/main/java/com/github/fribourgsdp/radio/ImportSpotifyPlaylistsActivity.kt:27]: No it is temporary.

File: app/src/main/java/com/github/fribourgsdp/radio/ImportSpotifyPlaylistsActivity.kt (hunk 2 above)
Comment (sim0mo) [reply app/src/main/java/com/github/fribourgsdp/radio/ImportSpotifyPlaylistsActivity.kt:129]: No this was not handled at all previously, that is the purpose of this PR, to do a provisonal solution. Nathan may be responsible for it in another sprint.

**Review by BartLucien — APPROVED** [review]
It is fine for me

**Review by shanzhaiii — APPROVED** [review]
I approve this as a temporary solution but in general it is a good idea to start to implement error handling in what we are loading from spotify

**Pull request merged**